### PR TITLE
feat(appeals): add change page for development description

### PR DESCRIPTION
**Pull request by AlexMoulin**
## Issue ticket number and link
https://pins-ds.atlassian.net/browse/A2-298
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/appeals/appeal-details/development-description/__tests__/development-description.test.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+const { app, installMockApi, teardown } = createTestEnvironment();
+const request = supertest(app);
+const baseUrl = `/appeals-service/appeal-details/${appealData.appealId}/appellant-case`;
+
+describe('development-description', () => {
+	beforeEach(installMockApi);
+	afterEach(teardown);
+
+	describe('/GET /change', () => {
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/development-description/__tests__/development-description.test.js:18]: Very minor, but the test names shouldn't have `/` before the method (eg. `GET /change` not `/GET /change`)

File: appeals/web/src/server/appeals/appeal-details/development-description/__tests__/development-description.test.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+		it('should re-render the development description change page with an error if the field is over 1000 char', async () => {
+			const apiCall = nock('http://test/')
+				.patch(`/appeals/${appealData.appealId}/appellant-cases/${appealData.appellantCaseId}`)
+				.reply(200, {});
+			const appellantCaseDataCall = nock('http://test/')
+				.get(`/appeals/${appealData.appealId}/appellant-cases/${appealData.appellantCaseId}`)
+				.reply(200, { appellantCaseDataIncompleteOutcome });
+
+			const invalidData = {
+				developmentDescription: `${'a'.repeat(1001)}` // Creating a 1001 char string.
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/development-description/__tests__/development-description.test.js:86]: the comment is probably unnecessary here

**Review by james-hill-kc — COMMENTED** [review]
Looks good! Just a couple of very minor comments from me

File: appeals/web/src/server/appeals/appeal-details/development-description/__tests__/development-description.test.js (hunk 2 above)
Comment (AlexMoulin) [reply appeals/web/src/server/appeals/appeal-details/development-description/__tests__/development-description.test.js:86]: Wasn't sure if anyone had seen repeat before :)

**Review by james-hill-kc — APPROVED** [review]
LGTM!

**Pull request merged**